doc/docs/1.10.x/enterprise/saml_setup.md

```diff
@@ -13,8 +12,9 @@ available in any official releases. This will describe the process of:
 
 ## Activation
 
-For testing, we **highly** recommend running Pachyderm in Minikube, in case any
-early bugs make it necessary to restart the cluster.
+For testing, we **highly** recommend running Pachyderm in Minikube, in case the
+process of configuring your cluster doesn't go well initially and you need to
```

Comment:
```suggestion
```
I recommend we delete this phrase because it suggests that something will inevitably go wrong.